mirrord/operator/src/setup.rs

```diff
@@ -314,6 +314,18 @@ impl OperatorDeployment {
             });
         }
 
+        let health_probe = Probe {
```

Comment:
you can just use https probe..

Reply:
There is no https probe, grpc and http only